### DISCUSSION TITLE
I need help downloading things

### DISCUSSION THREAD

**Discussion by Andresfane** [community/community#120314] [Accessibility]
### Select Topic Area

Question

### Body

It says downloads are disabled File uploads require access to this action, helpppp

**Comment by SiddhiShilkar** [comment]
If file uploads or downloads are disabled on GitHub:

1. Check repository settings.
2. Ensure permissions.
3. Verify file size and type.
4. Try a different network.
5. Clear browser cache.
6. Contact GitHub Support if issues persist.

**Comment by StackSorcerer403** [comment]
✔ Account Status: 
If GitHub has suspended your account, it can disable downloads. If that's the case, you'll need to wait for GitHub support to resolve it.

✔ Permissions Check: 
Ensure you have the necessary permissions in the repository. If not, contact the repository owner to gain access.

✔ Try Another Browser: 
Sometimes, browser extensions can cause issues. Switch to a different browser or disable extensions and see if that helps.

✔ Network Restrictions: 
If you're on a restricted network (like at work or school), try a different network. Your security software could also be blocking downloads.

✔ Contact GitHub Support: 
If the issue persists, reach out to GitHub support. Explain the problem clearly, including any error messages, and ask them to help you fix it.

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
